backend/github.js

```diff
@@ -0,0 +1,20 @@
+var exports = module.exports = {};
+var request = require("request");
+
+exports.retrieveOrgs = function (accessToken, callback) {
+  var organisations = [];
+  request({
```

Comment:
request.get can be used instead of directly using request with get as default method.